pycalphad/core/eqsolver.pyx

```diff
@@ -84,41 +82,43 @@ cdef bint add_new_phases(object composition_sets, object removed_compsets, objec
         return True
     return False
 
-# composition_sets: List[CompositionSet]
-# comps: List[v.Species]
-# cur_conds: OrderedDict[str, float]
-# iter_solver: SolverBase instance
-cpdef pointsolve(composition_sets, comps, cur_conds, iter_solver):
-    "Mutates composititon_sets with updated values if it converges. Returns SolverResult."
-    return _solve_and_update_if_converged(composition_sets, comps, cur_conds, Problem, iter_solver)
 
-cdef _solve_and_update_if_converged(composition_sets, comps, cur_conds, problem, iter_solver):
-    "Mutates composititon_sets with updated values if it converges. Returns SolverResult."
+cpdef update_composition_sets(composition_sets, solver_result, remove=True):
```

Comment:
Could the keyword argument be renamed to something more descriptive, like `remove_metastable`?